### ISSUE TITLE
Template not working for me

### ISSUE THREAD

**Issue by JimT787** [shannah/jdeploy-javafx-starter#2]
I tried following the exact directions on the Wiki for this repository, but it doesn't work--wondering if I'm doing something wrong.

I used the "Use this Template" button to "Create new repository" at JimT787/jim-jdeploy-test.

After clicking the "Create repository from this template" button, it did create the repository above, but nothing else happened even after several minutes.  After several minutes, I tried creating my own release with tag, but all it did was create some source code asset zip and tar files.   The wiki says it will create installer bundles.

It did not create any installer bundles for me.

Anything I'm doing wrong?  I have spent the last several days trying to figure out how to use jDeploy, and so far, I am unsuccessful and it looks like such a great solution, so thank you for creating it.

![Screenshot_20230209_114843](https://user-images.githubusercontent.com/98830368/217896688-dfe214eb-32a9-414a-a278-074bcfa2b4a7.png)

**Comment by shannah** [comment]
Thanks for reporting this.  There must have been a change to default permissions in github actions.  This used to work.  I have made a change to the github action in the template that should fix it.

You'll either need to create a new repo from the template, or copy the new workflow and package.json into your project from the template.

Relevant files:

- https://github.com/shannah/jdeploy-javafx-starter/blob/master/.github/workflows/jdeploy.yml
- https://github.com/shannah/jdeploy-javafx-starter/blob/master/package.json

**Comment by JimT787** [comment]
Thank you for the quick response! 

I re-downloaded the template and everything worked, however the built installers did not work.  I looked at the template files and found there is a difference between "sample" and "sample2" in the pom versus the package.json file.  The pom.xml file showed "sample" as the artifact, so I changed the package.json file to match by changing entries referencing "sample2" to "sample" and attached some images with relevant highlighted sections.  Maybe the template needs to be slightly changed?  Thanks, again. I look forward to using this great tool--very nice work!

![packagejson-highlighted](https://user-images.githubusercontent.com/98830368/217996672-4aa3b811-cf2c-4aad-a1da-3439f1ee1355.png)
![pom-highlighted](https://user-images.githubusercontent.com/98830368/217996674-7c8cdb7e-c2c4-4451-b250-8dc34aa9003b.png)
![packagejson-changed-and-improved](https://user-images.githubusercontent.com/98830368/217997325-2c8c338b-8fe1-4942-9865-7ac25640796d.png)
